lib/foreman_tasks_core/task_launcher/group.rb

```diff
@@ -0,0 +1,122 @@
+require 'foreman_tasks_core/runner'
+
+module ForemanTasksCore
+  module TaskLauncher
+    class AbstractGroup < Batch
```

Comment:
This is pretty complex class that would benefit from some documentation comment, explaining:

1. what's the responsibility of each sub-class in here (`GroupParentAction`, `Dummy`. `GroupRunner`, `BatchCallback`)
2. what's the interaction between the classes
3. what methods are expected to be overridden